cmds/core/dirname/dirname_test.go

```diff
@@ -29,27 +27,23 @@ var dirnameTests = []test{
 
 func TestDirName(t *testing.T) {
 	// Table-driven testing
+	var out = bytes.NewBuffer(nil)
+
 	for _, tt := range dirnameTests {
-		c := testutil.Command(t, tt.args...)
-		stdout, stderr := &bytes.Buffer{}, &bytes.Buffer{}
-		c.Stdout, c.Stderr = stdout, stderr
-		err := c.Run()
+		out.Reset()
+		err := run(out, tt.args)
+
 		if err != nil && tt.err == "" {
-			t.Errorf("Test %v: got %q, want nil", tt.args, err)
-			continue
+			t.Errorf("no error expected, got: \n%v", err)
```

Comment:
Then this code will be much simpler, you can use errors.Is I think?

Reply:
Yes, fixed in bfe3c18